apps/marketplace/components/validators.js

```diff
@@ -59,6 +63,16 @@ export const dateIs2DaysInFuture = val => {
   return false
 }
 
+export const startDateIsBeforeEndDate = (start, end) => {
+  // if (!validDate(start) || !validDate(end)) {
+  //   return false
+  // }
```

Comment:
What's the plan with this?  Delete?

Reply:
to return true if not valid.  valid check should be done elsewhere